instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/semconv/url/internal/InternalUrlAttributesExtractor.java

```diff
@@ -31,10 +32,12 @@ public void onStart(AttributesBuilder attributes, REQUEST request) {
     String urlScheme = getUrlScheme(request);
     String urlPath = getter.getUrlPath(request);
     String urlQuery = getter.getUrlQuery(request);
+    String urlTemplate = getter.getUrlTemplate(request);
 
     internalSet(attributes, UrlAttributes.URL_SCHEME, urlScheme);
     internalSet(attributes, UrlAttributes.URL_PATH, urlPath);
     internalSet(attributes, UrlAttributes.URL_QUERY, urlQuery);
+    internalSet(attributes, AttributeKey.stringKey("url.template"), urlTemplate);
```

Comment:
This constant is in the `UrlIncubatingAttributes` class. You will have to take a dependency on `opentelemetry-semconv-incubating` to reference that, and I suspect that the maintainers might frown on that. 😶

Reply:
I could not find any policies on stability, but I suspect you're right. (perhaps there is an expectation that things like this will be put behind a flag?)

Reply:
The current guidance is that incubating attribute classes such as `UrlIncubatingAttributes` should only be used in tests. Implementation needs to copy the constant because incubating attributes can be removed or rnamed in subsequent versions of the semantic conventions. You will have to move this code to incubating module, perhaps to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/http/HttpExperimentalAttributesExtractor.java